components/uid.rst

```diff
@@ -115,7 +115,7 @@ UUID objects created with the ``Uuid`` class can use the following methods
 Storing UUIDs in Databases
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-You can store UUID values as any other regular string/binary values in the database.
+You can store UUID values as any other regular string values in the database.
```

Comment:
arent they binary in database?

Reply:
Yes! I've reworded this while merging. Thanks.